code/game/jobs/job/special/cmb.dm

```diff
@@ -6,6 +6,11 @@
 /datum/job/special/cmb/marshal
 	title = JOB_CMB_TL
 
+AddTimelock(/datum/job/special/cmb/marshal, list(
```

Comment:
As far as I can tell neither of these timelocks actually talk to the emergency_call system. Instead it uses a number in emergency_call: time_required_for_job. It is default 5 hours and would need to be custom set, with both check_timelock() calls in the CMB emergency_call.

Reply:
All that to say, they're both currently timelocked for 5 hours for CL and SL, as far as I can tell, already